pkg/network/tracer/tracer.go

```diff
@@ -522,6 +522,9 @@ func (t *Tracer) getConnections(activeBuffer *network.ConnectionBuffer) (latestU
 		t.addProcessInfo(&activeConnections[i])
 	}
 
+	// get rid of stale process entries in the cache
+	t.processCache.Trim()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
As that a hot-code-path which is responsible for most of the OOMs in system-probe, I think we should run load test to see what's the impact on the memory and cpu

Reply:
Updated with graphs